### PR TITLE
doc: update 1.0.1 release history

### PR DESCRIPTION
**Pull request by frankqianms**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/apic-extension/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,15 @@
 Release History
 ===============
 
+1.0.1
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:6]: let bump the version to 1.1.0, as there is new feature

File: src/apic-extension/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,22 @@
...
 ===============
 
+1.1.0
+++++++++++++++++++
+Potential Impact: The changes in this release, including the renaming of commands and parameters, may require changes to existing scripts and integrations. Please review the changes carefully and update your code accordingly.
+
+**Updates:**
+* Added example for ``az apic api update`` command.
+* Added examples with system assigned identity for ``az apic create`` and ``az apic update`` commands.
+* [BREAKING CHANGE] Added ``--custom-metadata-only`` parameter to ``az apic metadata export`` command.
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:13]: it's not a breaking change

File: src/apic-extension/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,22 @@
...
+
+**Fixes:**
+* Resolved errors occurring when extracting external documentation from spec in ``az apic register`` command.
+
+**Removals:**
+* Eliminated duplicate example for ``az apic create `` command.
+* [BREAKING CHANGE] Removed the ``summary`` scope setting in ``az apic api register`` command.
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:20]: Let's move it to `fixes` section, as it's a bug per PM. Maybe something like: Do not use API description as summary in `az apic register` command.

File: src/apic-extension/HISTORY.rst [hunk 4]
```diff
@@ -3,6 +3,22 @@
...
+++++++++++++++++++
+Potential Impact: The changes in this release, including the renaming of commands and parameters, may require changes to existing scripts and integrations. Please review the changes carefully and update your code accordingly.
+
+**Updates:**
+* Added example for ``az apic api update`` command.
+* Added examples with system assigned identity for ``az apic create`` and ``az apic update`` commands.
+* [BREAKING CHANGE] Added ``--custom-metadata-only`` parameter to ``az apic metadata export`` command.
+
+**Fixes:**
+* Resolved errors occurring when extracting external documentation from spec in ``az apic register`` command.
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:16]: Let's make it more clear about what the error is: Set external document correctly in `az apic register` command.

File: src/apic-extension/HISTORY.rst [hunk 5]
```diff
@@ -3,6 +3,22 @@
 Release History
 ===============
 
+1.1.0
+++++++++++++++++++
+Potential Impact: The changes in this release, including the renaming of commands and parameters, may require changes to existing scripts and integrations. Please review the changes carefully and update your code accordingly.
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:8]: No need to add this paragraph as we don't rename commands or parameters.

File: src/apic-extension/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,21 @@
...
+++++++++++++++++++
+
+**Updates:**
+* Added example for ``az apic api update`` command.
+* Added examples with system assigned identity for ``az apic create`` and ``az apic update`` commands.
+* Added ``--custom-metadata-only`` parameter to ``az apic metadata export`` command.
+
+**Fixes:**
+* Set external document correctly in ``az apic register`` command.
+* [BREAKING CHANGE] Do not use API description as summary in ``az apic register`` command.
```
Comment (blackchoey) [inline src/apic-extension/HISTORY.rst:16]: Let's remove the breaking change label, as PM thinks setting description to summary is a bug.

Comment (blackchoey) [reply src/apic-extension/HISTORY.rst:16]: Other part looks good to me. Thaks!

Comment (frankqianms) [reply src/apic-extension/HISTORY.rst:16]: ok. thanks~

**Review by blackchoey — APPROVED** [review]

**Pull request merged**